### PR TITLE
Most notes

### PR DESCRIPTION
**Pull request by John151**
#8 Adds link to notes_list page to new most_notes page. most_notes shows in order concerts with most notes. Still need to clean up, for example it renders the band name "Name: Band name"
@Nelwell @andrea-pratt @abdiawali @PickleEric

### CODE REVIEW

File: lmn/models.py [hunk 1]
```diff
@@ -64,7 +64,9 @@ def save(self, *args, **kwargs):
         if old_note and old_note.image:
             if old_note.image != self.image:
                 self.delete_image(old_note.image)
-
+        # note_count = Notes.objects.all().count()
```
Comment (andrea-pratt) [inline lmn/models.py:67]: This should be cleaned up before merging. Everything's working for me though!

File: lmn/models.py [hunk 2]
```diff
@@ -87,6 +89,7 @@ def __str__(self):
 class Badge(models.Model):
     name = models.CharField(max_length=50, blank=False)
     description = models.CharField(max_length=200, blank=False)
+    # user = models.ForeignKey(Profile, on_delete=models.CASCADE)
```
Comment (andrea-pratt) [inline lmn/models.py:92]: Another line that just needs to be cleaned up.

**Review by andrea-pratt — CHANGES_REQUESTED** [review]
Looks good! Just a few commented-out lines of code that need to be addressed before we can merge.

File: lmn/models.py (hunk 1 above)
Comment (John151) [reply lmn/models.py:67]: You are correct. Changes made

**Comment by andrea-pratt** [comment]
Looks good!

**Pull request merged**